<launch>
    <!-- perception_bridge 节点 -->
    <node name="perception_bridge_two_node" pkg="perception_bridge" type="perception_bridge_two_node" output="screen">
        <!-- 加载参数文件 -->
        <rosparam command="load" file="$(find perception_bridge)/config/params.yaml" />
    </node>
</launch>
